core/models/postgres.py

```
@@ -0,0 +1,136 @@
+    version_id = Column(Integer, primary_key=True, autoincrement=True)
+    payload = Column(JSONB, nullable=False)
+    is_active = Column(Boolean, default=False)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    created_by = Column(String(50))
+    change_note = Column(Text)
+
+    __table_args__ = (
+        Index("idx_settings_active", "is_active", postgresql_where=(is_active == True)),
+    )
+
+    def __repr__(self):
```

Ruff (0.14.10)

Missing return type annotation for special method `__repr__`

Add return type annotation: `str`

(ANN204)